charts/datadog/templates/_container-otel-agent.yaml

```diff
@@ -3,10 +3,10 @@
   image: "{{ include "image-path" (dict "root" .Values "image" .Values.agents.image) }}"
   imagePullPolicy: {{ .Values.agents.image.pullPolicy }}
   {{- if eq .Values.targetSystem "linux" }}
-  command: ["otel-agent", "--config={{ template "datadog.otelconfPath" . }}/otel-config.yaml", "--core-config={{ template "datadog.confPath" .  }}/datadog.yaml"]
+  command: ["otel-agent", "--config={{ template "datadog.otelconfPath" . }}/otel-config.yaml", "--core-config={{ template "datadog.confPath" .  }}/datadog.yaml", "--sync-delay=30s"]
   {{- end -}}
   {{- if eq .Values.targetSystem "windows" }}
-  command: ["otel-agent", "-foreground", "-config={{ template "datadog.otelconfPath" . }}/otel-config.yaml", "--core-config={{ template "datadog.confPath" .  }}/datadog.yaml"]
+  command: ["otel-agent", "-foreground", "-config={{ template "datadog.otelconfPath" . }}/otel-config.yaml", "--core-config={{ template "datadog.confPath" .  }}/datadog.yaml", "--sync-delay=30s"]
```

Comment:
does this new parameter has been always available in the `otel-agent` ? 

If not, what happen if the `otel-agent` version doesn't support this parameter? is it only a warning or an error?

Reply:
Appears like this was added x.58: https://github.com/DataDog/datadog-agent/pull/28319. Although please note that the otel-agent is still in beta and the only public docs we have: https://docs.datadoghq.com/opentelemetry/agent/install_agent_with_collector/#configure-the-datadog-agent mention x.61.